web/src/main/java/org/openmrs/module/web/WebModuleUtil.java

```diff
@@ -408,83 +405,6 @@ public static boolean isModulePackageNameInTaskClass(String modulePackageName, S
 	}
 	
 	/**
-	 * Method visibility is package-private for testing
-	 *
-	 * @param mod
-	 * @param realPath
-	 * @should prefix messages with module id
-	 * @should not prefix messages with module id if override setting is specified
-	 */
-	static void copyModuleMessagesIntoWebapp(Module mod, String realPath) {
-		for (Entry<String, Properties> localeEntry : mod.getMessages().entrySet()) {
-			if (log.isDebugEnabled()) {
-				log.debug("Copying message property file: " + localeEntry.getKey());
-			}
-			
-			Properties props = localeEntry.getValue();
-			
-			if (!"true".equalsIgnoreCase(props
-			        .getProperty(ModuleConstants.MESSAGE_PROPERTY_ALLOW_KEYS_OUTSIDE_OF_MODULE_NAMESPACE))) {
```

Comment:
We definitely rely on defining keys outside the module namespace for i18n in Mirebalais (and it's generally the approach for the reference application). E.g. for this: https://github.com/PIH/openmrs-module-mirebalais/blob/master/api/src/main/resources/messages.properties#L184